OPHD/Technology/Technology.h

```diff
@@ -77,12 +77,20 @@ struct Technology
 		std::string value;
 	};
 
+
+	Technology() = delete;
+	Technology(int id, int labType, int cost) :
+		id(id),
+		labType(labType),
+		cost(cost)
+	{}

```

Comment:
Sure, I was going for aggregate initialization here, I just wanted to avoid having these instantiated without initialization. Again, coming a bit from C# and forgetting they're not the same language. Anyway I'm game for changing it. Will make updates.